contracts/peterMangrove/MgvOracle/app.yml

```diff
@@ -0,0 +1,9 @@
+name: mangrove-oracle
```

Comment:
Please change the dir name to also be `mangrove-oracle`. And for the username, you can just use `mangrove`

Reply:
Done